pyproject.toml

```diff
@@ -0,0 +1,45 @@
+# With this the user can use a custom
+# python environment, like those created
+# with venv or pyenv. The deault will be
+# the system python.
+[virtualenvs]
+prefer-active-python = true
+
+# This isnt really needed on pratical aspect
+# since this is only for organized the functional tests
+# but is necessary since its a requirement for poetry
+[tool.poetry]
+name = "floresta-functional-tests"
+version = "0.0.1"
+description = "collection of tools to help with functional testing of Floresta"
+authors = [
+  "lordshashank",
+  "qlrd <qlrddev@gmail.com>"
+]
+
+# Replace requirements.txt to here
+# if you wanna add some dep, just type `poetry add <package>`
```

Comment:
Replace with:

To add a new dependency, use `poetry add <package>`. To update, first find outdated packages with `poetry show --outdated`, then re-install with `poetry add <package>@<version>`

Reply:
fixed